client/state/jetpack-connect/selectors.js

```diff
@@ -25,15 +27,18 @@ const getFlowType = function( state, site ) {
  * XMLRPC errors can be identified by the presence of an error message, the presence of an authorization code
  * and if the error message contains the string 'error'
  *
- * @param state
- * @returns {Boolean}
+ * @param {object} state Global state tree
+ * @returns {Boolean} If there's an xmlrpc error or not
  */
 const hasXmlrpcError = function( state ) {
+	const authorizeData = state.jetpackConnect.jetpackConnectAuthorize;
+
 	return (
-		state.authorizeError &&
-		state.authorizeError.message &&
-		state.authorizationCode &&
-		state.authorizeError.message.indexOf( 'error' ) > -1
+		authorizeData &&
```

Comment:
This is just a matter of personal preference, but I find `_.get` syntax way more confusing that chaining a couple of truthiness checks... I guess that may change if/when I get used to it, but anyway I think it's way more 'js canonical' to just use an extra check ...

Even if it's a line longer, I find 
```
return (
	authorizeData.authorizeError.message &&
	authorizeData.authorizationCode &&
	authorizeData.authorizeError.message.indexOf( 'error' ) > -1
);
```
more readable than
``` 
return (
        authorizeData.authorizationCode &&
        includes( get( authorizeData, [ 'authorizeError', 'message' ] ), 'error' )
    );
```
The first one reads like prose, the second one reads like math :)